fluster/decoders/chromium.py

```diff
@@ -0,0 +1,64 @@
+# Fluster - testing framework for decoders conformance
+# Copyright (C) 2021, Fluendo, S.A.
+#  Author: Manuel Jimeno Martínez <mjimeno@fluendo.com>, Fluendo, S.A.
+#
+# This library is free software; you can redistribute it and/or
+# modify it under the terms of the GNU Library General Public
+# License as published by the Free Software Foundation; either
+# version 2 of the License, or (at your option) any later version.
+#
+# This library is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# Library General Public License for more details.
+#
+# You should have received a copy of the GNU Library General Public
+# License along with this library; if not, write to the
+# Free Software Foundation, Inc., 59 Temple Place - Suite 330,
+# Boston, MA 02111-1307, USA.
+# pylint: disable=import-outside-toplevel
+from abc import ABC
+
+from fluster.codec import Codec, OutputFormat
+from fluster.decoder import Decoder, register_decoder
+
+
+class ChromiumDecoder(Decoder, ABC):
+    """Generic class for Chromium decoder"""
+    description = "This is an implementation for the Chromium decoder"
+
+
+@register_decoder
+class ChromiumH264(ChromiumDecoder):
+    """H264 class for Chromium decoder"""
+    provider = 'Chromium'
+    codec = Codec.H264
+
+    def __init__(self) -> None:
+        super().__init__()
+        if not self.name:
+            self.name = f'{self.provider}-{self.codec.value}'
+        self.description = f'{self.provider} {self.codec.value} decoder for Chromium'
+
+    def decode(
+            self,
+            input_filepath: str,
+            output_filepath: str,
+            output_format: OutputFormat,
+            timeout: int,
+            verbose: bool,
+    ) -> str:
+        message_error = "Decoder doesn't available, module flusterchromium not installed"
+        try:
+            from flusterchromium import main  # type: ignore
+            return main(input_filepath)  # type: ignore
+        except RuntimeError as error:
+            return str(error)
+        except ModuleNotFoundError as error:
+            if verbose:
+                print(error)
```

Comment:
`print(error, file=sys.stderr)`

Reply:
ping @mjimenofluendo

Reply:
Why should this print appear in the stderr and this, being an error not?
https://github.com/fluendo/fluster/blob/e305fffe008570f7f4736564ac697dcdf0dfeeaa/fluster/decoder.py#L63
or this?
https://github.com/fluendo/fluster/blob/e305fffe008570f7f4736564ac697dcdf0dfeeaa/fluster/fluster.py#L154
And so, with all the error messages, shouldn't we keep a concordance with the rest of the code?